### PR TITLE
Update setup.md

### PR DESCRIPTION
**Pull request by kly-segment**

### CODE REVIEW

File: src/connections/reverse-etl/setup.md [hunk 1]
```diff
@@ -24,52 +24,38 @@ To add your warehouse as a source:
...
-After you add your data warehouse as a source, you can [add a model](#step-2-add-a-model) to your source.
-
 ## Step 2: Add a model
-Models are SQL queries that define sets of data you want to synchronize to your Reverse ETL destinations. After you add your source, you can add a model.
-
-> info "Use Segment's dbt extension to centralize model management and versioning"
-> Users who set up a BigQuery, Databricks, Postgres, Redshift, or Snowflake source can use Segment's [dbt extension](/docs/segment-app/extensions/dbt/) to centralize model management and versioning, reduce redundancies, and run CI checks to prevent breaking changes. 
->
-> Extensions is currently in public beta and is governed by Segment's [First Access and Beta Preview Terms](https://www.twilio.com/en-us/legal/tos){:target="_blank"}. During Public Beta, Extensions is available for Team and Developer plans only. [Reach out to Segment](mailto:friends@segment.com) if you're on a Business Tier plan and would like to participate in the Public Beta.
+Models define sets of data you want to sync to your Reverse ETL destinations. A source can have multiple models. Segment supports SQL models and dbt models.
```
Comment (kly-segment) [inline src/connections/reverse-etl/setup.md:36]: Hyperlink to respective headers for SQL & dbt

File: src/connections/reverse-etl/setup.md [hunk 2]
```diff
@@ -24,52 +24,38 @@ To add your warehouse as a source:
...
-
-Engage users can use the [Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/) to create and update [Profiles](/docs/unify/) that can then be accessed through [Profile API](/docs/unify/profile-api/) and activated within [Twilio Engage](/docs/engage). 
+Refer to the [Reverse ETL catalog](/docs/connections/reverse-etl/reverse-etl-catalog/) to view the supported actions destinations. Reverse ETL supports to unique destinations:
+- **[Segment Connections Destination](/docs/connections/reverse-etl/reverse-etl-catalog/#segment-connections-destination)**: Send warehouse data back into Segment to leverage your existing mappings or access non-actions destinations in the Connections catalog.
+- **[Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/)**: Engage users can use this destinaton to create and update [Profiles](/docs/unify/) that can then be accessed through [Profile API](/docs/unify/profile-api/) and activated within [Twilio Engage](/docs/engage).
```
Comment (kly-segment) [inline src/connections/reverse-etl/setup.md:58]: Could you rev this definition? Or copy what we provide in the public docs / internal docs?

File: src/connections/reverse-etl/setup.md [hunk 3]
```diff
@@ -93,39 +79,40 @@ To create a mapping:
 4. Select the model to sync from.
 5. Select the **Action** you want to sync and click **Next**.
       * Actions determine the information sent to the destination. The list of Actions will be unique to each destination.
-6. Add the mapping's name. The initial name will default to the Action's name (for example, 'Track Event') but is completely customizable. It will allow you to identify the mapping amongst others.
+6. Add the mapping name. The mapping name will default to the Action's name (e.g. 'Track Event'), but can be customized to allow you to identify the mapping across others.
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:82]: We avoid using Latin phrases (e.g., i.e., etc.) in the docs where possible.

```suggestion
6. Add a name for your mapping. The mapping name defaults to the Action's name (for example, 'Track Event'), but can be customized to allow you to identify the mapping across others.
```

File: src/connections/reverse-etl/setup.md [hunk 4]
```diff
@@ -93,39 +79,40 @@ To create a mapping:
...
 7. In the **Select record to map and send** section, select which records to send to your destination after Segment completes extracting data based on your model. You can choose from:
       * Added records
       * Updated records
       * Added or updated records
       * Deleted records
 8. Select a test record to preview the fields that you can map to your destination in the **Add test record** field.
-9. Select the Schedule type for the times you want the model’s data to be extracted from your warehouse. You can choose from:
-    * **Interval**: Extractions perform based on a selected time cycle.
+9. Select how often you want the schedule to sync in **Schedule configuration**.
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:89]:
```suggestion
9. Select how often you want Segment to sync your data under **Schedule configuration**.
```

File: src/connections/reverse-etl/setup.md [hunk 5]
```diff
@@ -81,10 +67,10 @@ To add your first destination:
 4. Select the Reverse ETL source you want to connect the destination to.
 5. Enter the **Destination name** and click **Create Destination**.
 6. Enter the required information on the **Settings** tab of the destination.
-7. Navigate to the destination settings tab and enable the destination. If the destination is disabled, then Segment won't be able to start a sync.
+7. Navigate to the destination settings tab and enable the destination. Note: If the destination is disabled, then Segment won't be able to start a sync.
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:70]:
```suggestion
7. Navigate to the destination settings tab and enable the destination. If the destination is disabled, then Segment won't be able to start a sync.
```

File: src/connections/reverse-etl/setup.md [hunk 6]
```diff
@@ -24,52 +24,38 @@ To add your warehouse as a source:
...
+### SQL editor
 1. Navigate to **Connections > Sources** and select the **Reverse ETL** tab. Select your source and click **Add Model**.
 2. Click **SQL Editor** as your modeling method. (Segment will add more modeling methods in the future.)
-3. Enter the SQL query that’ll define your model. Your model is used to map data to your Reverse ETL destinations.
+3. Enter the SQL query that’ll define your model. Your model is used to map data to your Reverse ETL destination(s).
 4. Choose a column to use as the unique identifier for each record in the **Unique Identifier column** field.
-    * The Unique Identifier should be a column with unique values per record to ensure checkpointing works as expected. It can potentially be a primary key. This column is used to detect new, updated, and deleted records.
-5. Click **Preview** to see a preview of the results of your SQL query. The data from the preview is extracted from the first 10 records of your warehouse.
+    * The Unique Identifier should be a column with unique values per record to ensure checkpointing works as expected, such as a primary key. This column is used to detect new, updated, and deleted records.
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:43]:
```suggestion
    * The Unique Identifier should be a column with unique values per record to ensure checkpointing works as expected, like a primary key. This column is used to detect new, updated, and deleted records.
```

File: src/connections/reverse-etl/setup.md [hunk 7]
```diff
@@ -24,52 +24,38 @@ To add your warehouse as a source:
 1. Navigate to **Connections > Sources** and select the **Reverse ETL** tab in the Segment app.
 2. Click **+ Add Reverse ETL source**.
 3. Select the source you want to add. 
-4. Follow the corresponding setup guide for your Reverse ETL source:
+4. Follow the corresponding guide to setup the required permissions for your Reverse ETL source:
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:27]:
```suggestion
4. Follow the corresponding guide to set up the required permissions for your Reverse ETL source:
```

File: src/connections/reverse-etl/setup.md [hunk 8]
```diff
@@ -24,52 +24,38 @@ To add your warehouse as a source:
...
-
-Engage users can use the [Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/) to create and update [Profiles](/docs/unify/) that can then be accessed through [Profile API](/docs/unify/profile-api/) and activated within [Twilio Engage](/docs/engage). 
+Refer to the [Reverse ETL catalog](/docs/connections/reverse-etl/reverse-etl-catalog/) to view the supported actions destinations. Reverse ETL supports to unique destinations:
+- **[Segment Connections Destination](/docs/connections/reverse-etl/reverse-etl-catalog/#segment-connections-destination)**: Send warehouse data back into Segment to leverage your existing mappings or access non-actions destinations in the Connections catalog.
+- **[Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/)**: Engage users can use this destinaton to create and update [Profiles](/docs/unify/) that can then be accessed through [Profile API](/docs/unify/profile-api/) and activated within [Twilio Engage](/docs/engage).
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:58]:
```suggestion
- **[Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/)**: Engage Premier Subscriptions users can use Reverse ETL to sync subscription data from their warehouses to destinations.
```

Comment (forstisabella) [reply src/connections/reverse-etl/setup.md:58]: This is what's in the RETL catalog page!

File: src/connections/reverse-etl/setup.md [hunk 9]
```diff
@@ -24,52 +24,38 @@ To add your warehouse as a source:
...
-After you add your data warehouse as a source, you can [add a model](#step-2-add-a-model) to your source.
-
 ## Step 2: Add a model
-Models are SQL queries that define sets of data you want to synchronize to your Reverse ETL destinations. After you add your source, you can add a model.
-
-> info "Use Segment's dbt extension to centralize model management and versioning"
-> Users who set up a BigQuery, Databricks, Postgres, Redshift, or Snowflake source can use Segment's [dbt extension](/docs/segment-app/extensions/dbt/) to centralize model management and versioning, reduce redundancies, and run CI checks to prevent breaking changes. 
->
-> Extensions is currently in public beta and is governed by Segment's [First Access and Beta Preview Terms](https://www.twilio.com/en-us/legal/tos){:target="_blank"}. During Public Beta, Extensions is available for Team and Developer plans only. [Reach out to Segment](mailto:friends@segment.com) if you're on a Business Tier plan and would like to participate in the Public Beta.
+Models define sets of data you want to sync to your Reverse ETL destinations. A source can have multiple models. Segment supports SQL models and dbt models.
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:36]:
```suggestion
Models define sets of data you want to sync to your Reverse ETL destinations. A source can have multiple models. Segment supports [SQL models](/docs/connections/reverse-etl/setup/#step-4-create-mappings) and [dbt models](/docs/segment-app/extensions/dbt/).
```

File: src/connections/reverse-etl/setup.md [hunk 10]
```diff
@@ -93,39 +79,40 @@ To create a mapping:
...
 7. In the **Select record to map and send** section, select which records to send to your destination after Segment completes extracting data based on your model. You can choose from:
       * Added records
       * Updated records
       * Added or updated records
       * Deleted records
 8. Select a test record to preview the fields that you can map to your destination in the **Add test record** field.
-9. Select the Schedule type for the times you want the model’s data to be extracted from your warehouse. You can choose from:
-    * **Interval**: Extractions perform based on a selected time cycle.
+9. Select how often you want Segment to sync your data under **Schedule configuration**.
+    * **Interval**: Extractions perform based on a selected time cycle. You can choose from: 15 minutes, 30 minutes, 1 hour, 2 hours, 4 hours, 6 hours, 8 hours, 12 hours, 1 day.
```
Comment (forstisabella) [inline src/connections/reverse-etl/setup.md:90]:
```suggestion
    * **Interval**: Extractions perform based on a selected time cycle. Select one of the following options: 15 minutes, 30 minutes, 1 hour, 2 hours, 4 hours, 6 hours, 8 hours, 12 hours, 1 day.
```

**Pull request merged**